### PR TITLE
Run some tests on GitHub Actions CI

### PR DESCRIPTION
**Pull request by anatoliykmetyuk**

### CODE REVIEW

**Comment by anatoliykmetyuk** [comment]
We can do the transition in several steps:

1. (This PR): Run test, test_bootstrapped and community_build on each push. Also run `sbt` and `java11` tests on tags (probably won't happen until next release). These tests will run on GitHub Actions machine, not ours. Hence this PR won't affect the performance or semantics of the existing CI. We'll be able to see if GH Actions do the right thing at the [Actions](https://github.com/lampepfl/dotty/actions) tab.
2. Release nightly builds from GitHub Actions. Also, generate the documentation for them.
3. Make Dotty releases a responsibility of GitHub Actions, transition their execution on our machines and ditch Drone CI altogether.

**Comment by nicolasstucki** [comment]
It looks like the community build workflow has an issue. https://github.com/dotty-staging/dotty/runs/445501828?check_suite_focus=true

**Comment by nicolasstucki** [comment]
Test output contains
```
[>                                      ] completed (53/1080, 0 failed, 10s)
[=>                                     ] completed (54/1080, 0 failed, 10s)
[=>                                     ] completed (54/1080, 0 failed, 11s)
[=>                                     ] completed (55/1080, 0 failed, 11s)
[=>                                     ] completed (56/1080, 0 failed, 11s)
[=>                                     ] completed (57/1080, 0 failed, 11s)
[=>                                     ] completed (58/1080, 0 failed, 11s)
[=>                                     ] completed (59/1080, 0 failed, 12s)
[=>                                     ] completed (59/1080, 0 failed, 12s)
[=>                                     ] completed (60/1080, 0 failed, 12s)
[=>                                     ] completed (62/1080, 0 failed, 12s)
[=>                                     ] completed (63/1080, 0 failed, 12s)
[=>                                     ] completed (63/1080, 0 failed, 13s)
[=>                                     ] completed (63/1080, 0 failed, 13s)
[=>                                     ] completed (64/1080, 0 failed, 13s)
```
we need to change the line width.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+          restore-keys: ${{ runner.os }}-general-
+
+      - name: Test
+        run: ./project/scripts/sbt sbt-dotty/scripted
+
+  test_java11:
+    runs-on: ubuntu-latest
+    container: akmetiuk/dotty:2020-02-12
+    if: github.event_name == 'push' &&
+        startsWith(github.event.ref, 'refs/tags/')
```
Comment (nicolasstucki) [inline .github/workflows/ci.yaml]: When will these run?

Comment (anatoliykmetyuk) [reply .github/workflows/ci.yaml]: When someone pushes a new tag

**Comment by anatoliykmetyuk** [comment]
So far community build is not included in the ci, needs a bit of debugging

**Pull request merged**